docs/core/deploying/trimming/snippets/ConsoleApp1/ConsoleApp1.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <PublishTrimmed>true</PublishTrimmed>
+    <PublishAot>true</PublishAot>
+    <IsTrimmable>true</IsTrimmable>
+    <InvariantGlobalization>true</InvariantGlobalization>
```

Comment:
What's the reason for `InvariantGlobalization=true`? It will have no effect on trim diagnostics. It is used by the AOT template to make the app smaller by default, but it has no effect on diagnostics (trim or AOT warnings).

Reply:
removed. The console template generated that.